angelsindustries/locale/en/logistic.cfg

```diff
@@ -26,7 +26,36 @@ angels-logistic-chest-buffer=Big buffer chest
 
 angels-crawler=Crawler construction vehicle
 
-angels-lamp=Crawler light
+angels-lamp=Crawler lamp
+
+[entity-description]
+cargo-robot=A slow logistic robot that can carry many items at once.
+cargo-robot-2=A slow logistic robot that can carry many items at once.
+cargo-roboport=This item should be hidden.
+cargo-hub=Early roboport with increased charging capabilities.
+cargo-box=Early roboport with large storage capacity for robots, but limited charging capabilities.
+angels-construction-robot=An early but slow construction robot.
+
+angels-relay-station=Extends the supply area of the logistic network and provides charging points for robots.
+angels-relay-station-2=Extends the supply area of the logistic network and provides charging points for robots.
+angels-relay-station-3=Extends the supply area of the logistic network and provides charging points for robots. Also provides minimal storage for robots.
+angels-charging-station=Provides charging points for robots with considerable recharge power.
+angels-zone-expander=Extends supply zone of the logistic network without requiring any electricity.
+angels-zone-expander-2=Extends supply zone of the logistic network without requiring any electricity.
+angels-zone-expander-3=Extends supply zone of the logistic network without requiring any electricity.
+angels-construction-zone-expander=Extends the construction zone of the logistic network.
+angels-construction-zone-expander-2=Extends the construction zone of the logistic network.
+angels-construction-zone-expander-3=Extends the construction zone of the logistic network. Also provides minimal charging for robots.
+
+angels-logistic-chest-passive-provider=Makes its content available to the logistic network.
+angels-logistic-chest-requester=Requests specified items from the logistic network.
+angels-logistic-chest-active-provider=Sends its content to the logistic network.
+angels-logistic-chest-storage=Long-term storage for the logistic network.
+angels-logistic-chest-buffer=Requests specified items to be available for personal logistics and automated construction.
+
+angels-crawler=Slow vehicle with huge trunk storage. Can be equipped with a vehicle roboport for mobile construction works.
+
+angels-lamp=Lamp with "Crawler" aesthetics.
```

Comment:
```suggestion
angels-lamp=A "Crawler"-graded Lamp safe for use in an industrial environment.
```

Reply:
"Crawler"-graded ? what is that supposed to mean?
It has the same health and still dies when the Crawler construction vehicle touches it -_-

Reply:
I can increase the health... but still, anything dies when a crawler passes over it :D (except rocks)